iothub_client/tests/iothubclient_amqp_e2e/iothubclient_amqp_e2e.c

```diff
@@ -5,6 +5,8 @@
 #include "iothubclient_common_e2e.h"
 #include "iothubtransportamqp.h"
 
+#define TELEMETRY_BATCH_SIZE 3500
```

Comment:
Does this mean we send 3500 messages?  That seems like a ton for a non-stress, non-longhaul run.  

Does it have to be this large?